/**:
  ros__parameters:
    use_sim_time: true
    # when this is set to "", the static_keyframe_provider will be a standalone node, otherwise it will be added to the specified container
    component_container_name: ""
    # when this is set to false, the static_keyframe_provider_node will be added to an existing container  
    create_own_container: false
    # x-y step size of the grid in which the static map keyframes are provided
    grid_step_size: 12.5
    # the default keyframe_radius = grid_step_size / sqrt(2), which means that the outer circle of a patch tangentially diagonally touch the neighboring keyframes
    # if this is set to a value > 0, the keyframe_radius will be set to this value to provide a larger or smaller overlap between the keyframes
    keyframe_radius: -1.0
    
    # the frame_id of the static keyframes
    frame_id: "map"

    # path to the pcd file which contains the static map
    # if the path is absolute (starting with /), the file will be loaded from the specified path
    # if the path is relative (starting without /), the file will be loaded from the install directory of the mrg_slam_static_keyframe_provider package
    pcd_path: "maps/marsyard2020_100000_points.pcd"
    # list of robots performing SLAM for providing static map keyframes in their vicinity, respective callbacks/services will be registered and called
    robot_names: ["atlas", "bestla"]
    # distance from robot to center of the static keyframe which are going to be provided
    slam_distance: 20.0 

    # voxel grid filter
    enable_voxel_grid_filter: true
    voxel_grid_resolution: 0.3

    # the frequency at which the static map provider publishes the static map keyframes if service is called
    timer_frequency: 3.0